content/en/docs/test.md

```diff
@@ -62,26 +62,30 @@ culpa qui officia deserunt mollit anim id est laborum.
 - This is a list item
 * This is another list item in the same list
 - You can mix `-` and `*`
-  - To make a sub-item, indent two spaces.
-    - This is a sub-sub-item.
+    - To make a sub-item, indent two tabstops (4 spaces). **This is different
+      from Jekyll / Kramdown.**
+        - This is a sub-sub-item. Indent two more tabstops (4 more spaces).
+    - Another sub-item.
 
+<!-- separate lists -->
 
-
-- This is a new list. It has two blank lines from the previous list, If it only
-  had one blank line, it would still show up as part of the previous list.
+- This is a new list. With Hugo, you need to use a HTML comment to separate two
+  consecutive lists. **The HTML comment needs to be at the left margin.**
 - Bullet lists can have paragraphs or block elements within them.
 
-  Just indent the content to be even with the text of the bullet point, rather
-  than the bullet itself.
+      Just indent the content to be one tab stop beyond the text of the bullet
```

Comment:
Why do we have to do these extra levels of indentation, @bep? This is very non-intuitive and hard to even describe adequately.